lib/is_external_link.js

```diff
@@ -18,17 +10,19 @@ const urlObj = (str) => {
 
 function isExternalLink(url) {
   const { config } = this;
-  const exclude = Array.isArray(config.external_link.exclude) ? config.external_link.exclude
-    : [config.external_link.exclude];
-  const data = urlObj(url);
-  const host = data.hostname;
-  const sitehost = typeof urlObj(config.url) === 'object' ? urlObj(config.url).hostname : config.url;
+  const sitehost = parse(config.url).hostname || config.url;
+  // Pass a base to new URL
+  const data = new URL(url, `http://${sitehost}`);
 
   if (!sitehost || typeof data === 'string') return false;
```

Comment:
with L15, `typeof data` is no longer required since it will always be an "object".

Reply:
Removed.